### ISSUE TITLE
Import %>% function

### ISSUE THREAD

**Issue by amstilp** [UW-GAC/dbgaptools#19]
I get an error when I run `check_pheno` if I don't load some other tidyverse packages too. Here's some sample code:
```r
library(dbgaptools)
subj_file <- "SAS/Subject_20181214.txt"
pheno_file <- "topmed_dcc_baseline_common_covariates_v2.txt"
dd_file <- "topmed_dcc_baseline_common_covariates_v2_DD.txt"
check_pheno(pheno_file, ddfile = dd_file)
```
And here's the error message:
```
Error in encoded_vars %<>% dplyr::mutate_all(function(x) stringr::str_replace(x,  : 
  could not find function "%>%"
```
I can fix it by loading one of the packages that exports `%>%`, but it should be imported in this package so it runs without having to load any additional packages.

**Comment by sarahcn** [comment]
Thanks for the heads up. So I need to import magrittr?

**Comment by smgogarten** [comment]
Fixed in 17583a7
